src/js/core/widget/core/SectionChanger.js

```diff
@@ -241,7 +241,7 @@
 						self._initLayout();
 						self._super(element);
 						self._repositionSections(true);
-						self.setActiveSection(self.activeIndex);
+						self.setActiveSection(self.activeIndex, false);
```

Comment:
You were going to add "false" as default value to 'direct' but you set default value for duration. Please check source code for `setActiveSection`

Reply:
You can set default values for method like: 
```
				setActiveSection: function (index, duration, direct) {
					var position = this.sectionPositions[index],
						scrollbarDuration = duration || 0,
						oldActiveIndex = this.activeIndex,
						newX = 0,
						newY = 0;

					direct = !!direct;
```

Reply:
done